packages/rspack/tests/configCases/builtins/minify-keep-classnames/main.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs");
+const path = require("path");
+
+it("should keep class names", () => {
+	const content = fs.readFileSync(path.resolve(__dirname, "entry.js"), "utf-8");
+	expect(content).toContain("class KeepClass");
```

Comment:
Here we can use the name field of the class

Reply:
fixed